### PR TITLE
[NUI] Add color constructor

### PR DESCRIPTION
**Pull request by neostom432**
### Description of Change ###
Add color constructor which is initialized by System.Drawing.Color.

When we test something, we always feel our Color class has not enough
color list.

In System.Drawing.Color, there is a ton of predefined colors.
So, added new constructor which can initialized by System.Drawing.Color.

### API Changes ###
NONE

### CODE REVIEW

File: src/Tizen.NUI/src/public/Color.cs [hunk 1]
```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
             }
         }
 
+        /// <summary>
+        /// The conversion constructor from an System.Drawing.Color instance.
+        /// </summary>
+        /// <param name="color">System.Drawing.Color instance</param>
+        /// <since_tizen> 6 </since_tizen>
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Color.cs:148]: please remove <since_tizen> tag if this is hidden api.

File: src/Tizen.NUI/src/public/Color.cs [hunk 2]
```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
...
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Color.cs:149]: please change "tizen_6.0" as "tizen_next" if this API's open schedule is not confirmed.

File: src/Tizen.NUI/src/public/Color.cs [hunk 3]
```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
...
+            try
+            {
+                R = color.R / 255.0f;
+                G = color.G / 255.0f;
+                B = color.B / 255.0f;
+                A = color.A / 255.0f;
+            }
+            catch
+            {
+                throw new global::System.ArgumentException("Please check your System.Drawing.Color instance");
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Color.cs:162]: could you check the following link?
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1065?view=vs-2019
even though this is not static constructor, if it throws exception in constructor, user could get some big confusion, so is there any other option to handle error case? (for example, giving a empty object or null ...)
I am not sure about this, so I am asking for your idea... ^^;

File: src/Tizen.NUI/src/public/Color.cs [hunk 4]
```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
             }
         }
 
+        /// <summary>
+        /// The conversion constructor from an System.Drawing.Color instance.
+        /// </summary>
+        /// <param name="color">System.Drawing.Color instance</param>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Color.cs:150]: if the api has exception, the comment needs show the description of the exception. 
example) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69
could you add some exception comment?

File: src/Tizen.NUI/src/public/Color.cs (hunk 1 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/Color.cs]: DONE

File: src/Tizen.NUI/src/public/Color.cs (hunk 2 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/Color.cs]: DONE

File: src/Tizen.NUI/src/public/Color.cs (hunk 4 above)
Comment (neostom432) [reply src/Tizen.NUI/src/public/Color.cs:148]: Remove exception because if color instance has problem, crash should be happened before.

**Review by dongsug-song — APPROVED** [review]
This is very good idea to use .NET's Color class...
How about adding "implicit conversion" for NUI.Color class and Drawing.Color class?
(for example, currently in NUI, Size and Size2D are converting each other implicitly.)
I will figure it out if this is possible...

**Pull request merged**